files/matreshka/update.json

```diff
@@ -3,6 +3,13 @@
   "name": "matreshka",
   "repo": "matreshkajs/matreshka",
   "files": {
-    "include": ["matreshka.min.js", "matreshka.js", "matreshka.min.map"]
+    "include": [
+		"matreshka.min.js",
+		"matreshka.js",
+		"matreshka.min.map",
+		"magic/matreshka-magic.min.js",
```

Comment:
Since you know the name you can just create the URL manually https://github.com/jsdelivr/jsdelivr/tree/master/files/matreshka